extensions/go/README.md

```diff
@@ -228,7 +249,50 @@ spec:
                   name: cache-ssd
 ```
 
-## Private dependencies
+With a corresponding service:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+    annotations:
+        prometheus.io/port: '6060'
+        prometheus.io/scrape: 'true'
+    labels:
+        app: lang-go
+    name: lang-go
+    namespace: prod
+spec:
+    loadBalancerIP: your.static.ip.address
+    ports:
+        - name: debug
+          port: 6060
+          targetPort: debug
+        - name: lsp
+          port: 443
+          targetPort: lsp
+    selector:
+        app: lang-go
+    type: LoadBalancer
+```
+
+#### TLS
+
+TLS is optional but recommended for production deployments. It is used if `TLS_KEY` and `TLS_CERT` environment variables are set.
```

Comment:
```suggestion
To enable TLS, set the `TLS_KEY` and `TLS_CERT` environment variables. TLS optional but **strongly recommended** for production deployments.
```

Is `TLS_CERT` a path or an encoded certificate?

Reply:
Looks like an encoded certificate based on the config in https://github.com/sourcegraph/deploy-sourcegraph-dot-com.